//package com.sincetimes.redwar.game.comnunicate.protocol;
message RseObtainPayUrlParame
{
	optional string url_params = 1;
}